Calculator/Calculator/Model/Extension/String.swift

```diff
@@ -0,0 +1,8 @@
+//  String.swift
+//  Created by 레옹아범 on 2023/01/26.
+
+extension String: CalculateItem {
+    func split(with target: Character) -> [String] {
+        return self.split(separator: target).map{ String($0) }
+    }
```

Comment:
고차함수와 중괄호 `{` 사이에는 띄어쓰기를 넣어주세요!
```suggestion
    func split(with target: Character) -> [String] {
        return self.split(separator: target).map { String($0) }
    }
```